src/cmd/backup.go

```diff
@@ -48,7 +48,7 @@ func Backup(spicetifyVersion string) {
 	if totalApp > 0 {
 		utils.PrintGreen("OK")
 	} else {
-		utils.PrintError("Cannot backup app files. Reinstall Spotify and try again.")
+		utils.PrintError("Cannot backup app files. Does Spicetify have permissions to access Spotify files?")
```

Comment:
So, unfortunately, there are more cases where this message occurs, so I do not feel like it matches all cases. Overall the current message targets most cases, therefore we should keep it for now.

Reply:
Understood, thanks for your response.